src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -1359,5 +1385,10 @@ public static IEnumerable<BlittableJsonReaderObject> GetAttachmentsFromDocumentM
                 }
             }
         }
+
+        [StorageIndexEntryKeyGenerator]
+        private static ByteStringContext.Scope GenerateBucketAndEtagIndexKeyForAttachments(ByteStringContext context, ref TableValueReader tvr, out Slice slice) =>
+            ExtractIdFromKeyAndGenerateBucketAndEtagIndexKey(context, (int)AttachmentsTable.LowerDocumentIdAndLowerNameAndTypeAndHashAndContentType,
+                (int)AttachmentsTable.Etag, ref tvr, out slice);
```

Comment:
Let's make this a real function with `{}` - makes it easier to read.

Reply:
done